targets/support/kmerge.sh

```diff
@@ -160,7 +160,8 @@ if [[ ${distkernel} = "yes" ]] ; then
   # Kernel already built, let's run dracut to make initramfs
   distkernel_source_path=$(equery -Cq f ${ksource} | grep "/usr/src/linux-" -m1)
   distkernel_image_path=$(distkmerge_get_image_path)
-  distkernel_version=${distkernel_source_path##"/usr/src/linux-"}
+  distkernel_mod_dir=$(ls -dt /lib/modules/* | head -1) # get the most recently modified kernel
```

Comment:
This is not necessarily the correct path.  You should use equery as above to get the correct path for the specified kernel.

Reply:
Hi sorry, this got superseded by kernel-* eclass changes. I'll close this PR now :)